corehq/apps/sms/api.py

```diff
@@ -153,9 +154,14 @@ def send_sms(domain, contact, phone_number, text, metadata=None, logged_subevent
     if domain and contact and is_commcarecase(contact):
         backend_name = contact.get_case_property('contact_backend_id')
         backend_name = backend_name.strip() if isinstance(backend_name, str) else ''
+
         if backend_name:
             try:
                 backend = SQLMobileBackend.load_by_name(SQLMobileBackend.SMS, domain, backend_name)
+
+                if not _backend_enabled_for_domain(backend, domain):
+                    return False
```

Comment:
what happens after this? Does it get logged anywhere?

Reply:
In that case I could just `raise BadSMSConfigException` instead of returning, since the try-except is already catching the `BadSMSConfigException` error.